File: Examples/CSharp/Programming-Documents/Fields/FormFieldsWorkWithProperties.cs
using System;
using System.Collections;
using System.IO;
using Aspose.Words;
using Aspose.Words.Fields;
namespace Aspose.Words.Examples.CSharp.Programming_Documents.Working_with_Fields
{
    class FormFieldsWorkWithProperties
    {
        public static void Run()
        {
            //ExStart:FormFieldsWorkWithProperties
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithFields();

            Document doc = new Document(dataDir + "FormFields.doc");
            FormField formField = doc.Range.FormFields[3];

            if (formField.Type.Equals(FieldType.FieldFormTextInput))
                formField.Result = "My name is " + formField.Name;
            //ExEnd:FormFieldsWorkWithProperties            
        }
    }
}

File: Examples/VisualBasic/VisualBasic/Programming-Documents/Fields/FormFieldsWorkWithProperties.vb
Imports Microsoft.VisualBasic
Imports System.IO
Imports Aspose.Words
Imports Aspose.Words.Fields
Public Class FormFieldsWorkWithProperties
    Public Shared Sub Run()
        ' ExStart:FormFieldsWorkWithProperties
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithFields()

        Dim doc As New Document(dataDir & Convert.ToString("FormFields.doc"))
        Dim formField As FormField = doc.Range.FormFields(3)

        If formField.Type.Equals(FieldType.FieldFormTextInput) Then
            formField.Result = "My name is " + formField.Name
        End If
        ' ExEnd:FormFieldsWorkWithProperties            
    End Sub
End Class
